### ISSUE TITLE
Can we promote your plugin on the main spotless homepage?

### ISSUE THREAD

**Issue by nedtwigg** [moznion/sbt-spotless#22]
Hi!  This looks great!  I am a Spotless maintainer, and I only bumped into this by searching "spotless" into `https://search.maven.org/` as I tried to figure out a publishing hangup.

In the [root Spotless readme](https://github.com/diffplug/spotless/blob/main/README.md), we have big bold sections "Spotless for Gradle", "Spotless for Maven", along with a row for each step, and a column for each build tool.  Can we link out to this project, to let people know that you're here?

I don't know the `sbt` conventions at all, but we recently adopted a common readme format across the [gradle](https://github.com/diffplug/spotless/tree/main/plugin-gradle) and [maven](https://github.com/diffplug/spotless/tree/main/plugin-maven) plugins. The format has made our readme maintenance a lot easier - we can always figure out where to put things now, and it's easy to generate durable permalinks.

If you'd like, now or in the future, I'd be happy to add you as a committer to diffplug/spotless, and we can move your plugin in-tree.  The advantages to that would be that it would be easier for your users to see when something got fixed.  Each time we fix an issue (e.g. https://github.com/diffplug/spotless/issues/668), we add a little callout with which plugin version has that fix.  The disadvantage is that we'd have to figure out how to mush gradle and sbt together.  Doing that for the maven plugin was a little bit painful.  Happy to help you do that, or happy to just link out to this repo.  Or if you prefer peace and quiet, that's cool too :)

Anyway, cool plugin, let me know if we can help you from upstream :)

**Comment by moznion** [comment]
@nedtwigg 
hi, thank you for contacting me.

> In the root Spotless readme, we have big bold sections "Spotless for Gradle", "Spotless for Maven", along with a row for each step, and a column for each build tool. Can we link out to this project, to let people know that you're here?

Yes, of course. I'll be happy if you add this plugin to the document!

> If you'd like, now or in the future, I'd be happy to add you as a committer to diffplug/spotless, and we can move your plugin in-tree. The advantages to that would be that it would be easier for your users to see when something got fixed. Each time we fix an issue (e.g. diffplug/spotless#668), we add a little callout with which plugin version has that fix. The disadvantage is that we'd have to figure out how to mush gradle and sbt together. Doing that for the maven plugin was a little bit painful. Happy to help you do that, or happy to just link out to this repo. Or if you prefer peace and quiet, that's cool too :)

This sounds also great, I'd like to be on-board this plan if your team accepts me and the plugin :)
What are the needed actions for this?

**Comment by nedtwigg** [comment]
I added a PR which links to this project, does this look right to you?

https://github.com/diffplug/spotless/pull/689/commits/5cd0eede65538ea4c65fa7214d5427542bc2efa2?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

We can discuss implementation details further with the full spotless community at the link above.  I think the only hiccup will be getting the build running with Gradle.  If you really don't want to build the sbt plugin with gradle, we can maintain the buildscripts in parallel.  But the publishing and changelog mechanisms we have setup rely on Gradle.  I'm pretty slammed for the next few weeks, but I'd be happy to help with Gradle parts if you're not familiar with it.  Whenever we merge a really big PR from someone, we give them commit access: https://openopensource.org/

**Comment by moznion** [comment]
The pull-request seems perfect! Thank you for doing this.

And I usually use the gradle on my job, so I think I can build a sbt plugin with the gradle.
I’d like to try porting this plugin to under the root project. When finished, can I send a pull-request to the root project?

**Comment by nedtwigg** [comment]
Yes please!  And feel free to open an incomplete PR, we'd be happy to help finish it with you.

**Comment by moznion** [comment]
Thank you, I'm going to work on that 👍
